UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles.cs

```diff
@@ -214,7 +214,8 @@ private string GetMimeType(string fileName, string mimeType, string batchId, str
             }
         }
 
-        public async Task<bool> CreateProductFile(string batchId, string exchangeSetInfoPath, string correlationId, SalesCatalogueDataResponse salesCatalogueDataResponse, DateTime scsRequestDateTime)
+        //encryption = true only for S63 AIO and Large Media Exchange Set
```

Comment:
Yes, this comment is required. In case of Large Media and AIO exchange set, service will only create s63 data.
To avoid ad hoc flag change value, we have added this comment.